### PR TITLE
kubeadm-1.15-certs-renewal

### PR DESCRIPTION
**Pull request by fabriziopandini**
Changes content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md for v1.15

@kubernetes/sig-cluster-lifecycle-pr-reviews
/cc @neolit123 @jimangel 
/sig cluster-lifecycle
/milestone v1.15

### CODE REVIEW

**Comment by neolit123** [comment]
xref https://github.com/kubernetes/kubeadm/issues/1577

**Comment by jimangel** [comment]
Preview: https://5cf6680028f0d20008f670aa--kubernetes-io-vnext-staging.netlify.com/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -7,46 +7,106 @@ content_template: templates/task
...
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
-These are advanced topics for users who need to integrate their organization's certificate infrastructure into a kubeadm-built cluster. If kubeadm with the default configuration satisfies your needs, you should let kubeadm manage certificates instead.
+Be familiar with [PKI certificates and requirements in Kubernetes](/docs/setup/certificates/).
 
-You should be familiar with [PKI certificates and requirements in Kubernetes](/docs/setup/certificates/).
+Have a working Kubernetes cluster installed using kubeadm with your certificates stored in `/etc/kubernetes/pki` folder; in case you are using a different location, the following can be adapted accordingly.
```
Comment (jimangel) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: I would suggest a pre-req that requires "kubeadm 1.15 or later," alternately you can add in, at the top, something like:
```
{{< feature-state for_k8s_version="v1.15" state="stable" >}}
```

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 2]
```diff
@@ -7,46 +7,106 @@ content_template: templates/task
...
+apiserver-etcd-client      May 15, 2020 13:00 UTC   364d            false
+apiserver-kubelet-client   May 15, 2020 13:00 UTC   364d            false
+controller-manager.conf    May 15, 2020 13:03 UTC   364d            false
+etcd-healthcheck-client    May 15, 2020 13:00 UTC   364d            false
+etcd-peer                  May 15, 2020 13:00 UTC   364d            false
+etcd-server                May 15, 2020 13:00 UTC   364d            false
+front-proxy-client         May 15, 2020 13:00 UTC   364d            false
+scheduler.conf             May 15, 2020 13:03 UTC   364d            false
+```
+
```
Comment (jimangel) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: We should include  a note snippet or block that states: "Kubelet.conf is managed by kubelet itself (kubeadm configures kubelet for automatic certificate renewal)" or similar

Comment (fabriziopandini) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: Done, at the end of the paragraph

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md (hunk 1 above)
Comment (fabriziopandini) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: Done, at the beginning of the doc

**Comment by fabriziopandini** [comment]
@jimangel everything should be addressed now!
Thanks again for the support

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 3]
```diff
@@ -7,46 +7,112 @@ content_template: templates/task
...
+## Automatic certificate renewal
+
+`kubeadm` renews all the certificates during control plane [upgrade](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15/).
+
+This feature is designed for addressing the simplest use cases; 
+if you don't have specific requirements on certificate renewal and perform Kubernetes version upgrades regularly (less than 1 year in between each upgrade), kubeadm will take care of keeping your cluster up to date and reasonably secure.
+
+{{< note >}}
+It is a best practice to upgrade your cluster frequently in order stay secure.
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: `to stay secure`

**Comment by jimangel** [comment]
nit: we might want to consider standardizing the order of notes / warnings / caution (for example the top one goes warning -> note, later we do note -> warning).

**Comment by fabriziopandini** [comment]
typo and note/warning order fixed!

**Comment by tengqm** [comment]
/lgtm
/approve

Jim's feedback has been addressed, good to go now.

**Pull request merged**